src/core/gltf2exporter/gltf2exporter_p.h

```diff
@@ -57,7 +57,7 @@ class SceneEntity;
 class KUESA_PRIVATE_EXPORT GLTF2ExportConfiguration
 {
 public:
-    enum MeshAttribute {
+    enum MeshAttribute : int8_t {
```

Comment:
why the enum class?

Reply:
sorry, do you mean the int8_t or the lack of `enum class` ? if that's the first I just prefer to be more explicit about it (I doubt there are going >100 kinds of attributes for meshes) but it's not that important.

Reply:
let as simple `enum` for now